src/bin/conserve.rs

```diff
@@ -180,16 +188,26 @@ fn init(subm: &ArgMatches, _report: &Report) -> Result<()> {
 
 
 fn cmd_backup(subm: &ArgMatches, report: &Report) -> Result<()> {
-    BackupOptions::default()
-        .backup(Path::new(subm.value_of("archive").unwrap()),
-                Path::new(subm.value_of("source").unwrap()),
-                report)
+    match subm.value_of("exclude") {
+        Some(exclude) => {
+            BackupOptions::with_excludes(Some(exclude.split(',').collect()))?
+                .backup(Path::new(subm.value_of("archive").unwrap()),
+                        Path::new(subm.value_of("source").unwrap()),
+                        report)
+        }
+        None => {
+            BackupOptions::default()
```

Comment:
For example here, if you make `with_excludes` modify an existing `BackupOptions`, you won't need to repeat so much code.